docs/signals.rst

```diff
@@ -50,3 +50,71 @@ Arguments sent with this signal:
 ``rework``
     A :class:`bool` value specifying if this was the first time the
     collection was submitted, or a rework of its moderation requests
+
+
+
+``published``
+------------------------
+
+.. attribute:: djangocms_moderation.submitted_for_review
+   :module:
+
+.. ^^^^^^^ this :module: hack keeps Sphinx from prepending the module.
+
+Sent when a :ref:`moderation_collection` is being published
+
+Arguments sent with this signal:
+
+``sender``
+    :class:`djangocms_moderation.models.ModerationCollection` class
+
+``collection``
+    A :class:`djangocms_moderation.models.ModerationCollection` instance
+    which was submitted to be published.
+
+``moderator``
+    A :class:`django.contrib.auth.models.User` associated with the collection which is the moderator of the collection.
+
+``moderation_requests``
+    A list of :class:`djangocms_moderation.models.ModerationRequest` instances
+    which were published.
+
+    .. note::
+
+        It's possible for this list to contain only some of the requests
+        belonging to the collection being moderated,
+        because only some of the requests were published.
+
+``workflow``
+    An instance of :class:`djangocms_moderation.models.Workflow` which was used for this collection.
+
+    .. note::
+
+        The workflow determines if the collection is being published or unpublished.
+
+
+How to use the moderation publish signal for a collection
+---------------------------------------------------------------------
+
+The CMS used to provide page publish and unpublish signals which have since been removed in DjangoCMS 4.0. You can instead use the signals provided above to replace these.
+
+Djangocms-moderation provides a way to take further actions once a collection has been published. The `published` event is the last event executed for a moderation.
+
+
+.. code-block:: python
+
+    from django.dispatch import receiver
+
+    from cms.models import PageContent
+
+    from djangocm_moderation.signals import published
+
+
+    @receiver(published)
+    def do_something_unpublish_event(*args, **kwargs):
```

Comment:
This should be "publishing" rather than "unpublishing"

Reply:
I'll change this to `do_something_on_published_event` 

Related to #166, do we want to have a separate event trigger for unpublished? For a user of the library it would be clearer, technically you would be able to achieve the same by checking whether the workflow is `is_unpublishing` or not.

Reply:
Personally I'd go for separate signals as they are separate concerns. We should stay consistent with Versioning.